### DISCUSSION TITLE
Flap sensitivity still too high?

### DISCUSSION THREAD

**Discussion by Hansvanwijhe** [flybywiresim/aircraft#3836] [Help]
Even after the latest Asobo patch it seems to me that when I set flaps to 1 during my descend, the responsivity of this action is way too high. The plane (CFBW) starts to climb at +/- 2000ft. What can be done here? Should I half the lift_coef_flaps in flightmodel.cfg to 1.2 (it now is 2.4) or even lower to, say, 0.6?

Thanks,
Hans

**Comment by CptBayer** [comment]
same observations here... I didnt´get an answer till now ... but I think, it is a known issue and the guys looking for a solution :-p ...


greets Klaus

**Reply by Hansvanwijhe** [reply]
Thanks, Klaus. Good to know I am not the only one experiencing this phenomenon. 

Best regards,
Hans
